runtime/lua/vim/lsp.lua

```diff
@@ -4,15 +4,15 @@ local lsp_rpc = require('vim.lsp.rpc')
 local protocol = require('vim.lsp.protocol')
 local util = require('vim.lsp.util')
 local sync = require('vim.lsp.sync')
-local api = vim.api
 
 local vim = vim
+local a = vim.api
 local nvim_err_writeln, nvim_buf_get_lines, nvim_command, nvim_buf_get_option, nvim_exec_autocmds =
-  vim.api.nvim_err_writeln,
-  vim.api.nvim_buf_get_lines,
-  vim.api.nvim_command,
-  vim.api.nvim_buf_get_option,
-  vim.api.nvim_exec_autocmds
+  a.nvim_err_writeln,
+  a.nvim_buf_get_lines,
+  a.nvim_command,
+  a.nvim_buf_get_option,
+  a.nvim_exec_autocmds
 local uv = vim.loop
```

Comment:
What's the general opinion on "hiding" `vim.loop` like that? It's shorter, and points towards the Luv docs, but on the other hand breaks the link to `:h vim.loop`.

Reply:
no objection, but consistency in our internal code is very very nice, so thank you @ii14